### ISSUE TITLE
delete/rename/move current file

### ISSUE THREAD

**Issue by svefro** [mifi/lossless-cut#87]
First of all, this is a great application and one that i have looked for for a long time.
i have accumulated a-lot of video over the years that contains alot of useless matereal.
i have started a clean-up now and want to cut out all that i want to keep and delete the rest.

some files can be deleted, some i want to keep and others require some cutting.

in this process it would be great with a delete/rename/move function in this application.

this way when clipping is done
-the file could be deleted from disk at once.
-renamed to something like "full_somefile.mpg" from "somefile.mpg"or "processed_somefile.mpg"
-moved to a subfolder called "processed" / "originals"

**Comment by mifi** [comment]
Can't you just do this manually? or with some simple bash lines

**Comment by svefro** [comment]
Yes but the whole point it to make it easier to clean up a large media collection.
Work flow:
Drag inn file, determine if something is useful in it.
Either:
-keep whole file.
-cut out the wanted parts, then delete the original.
-delete the file entirely.
drag inn next file and repeat.

i have had a look at the code and i think this should be fairly quick to implement.
I'm new to chromium desktop applications using electronjs and i have not had the time to get a full overview off the application yet.

Mousetrap.bind('r', () => this.renameFile());
Then a function that closes the current file and then renames it with a suffix.

This way all files with that suffix could be deleted at once when everything is done.

**Comment by mifi** [comment]
How about an "open in finder/explorer" shortcut or button, then the user can do what they want with the file? i think deleting is a bit scary.

**Comment by svefro** [comment]
Personally "open in finder/explorer" want do it for me since explorer is already opened in the current folder where i opened the file.

I agree that deleting it is scary.
A rename should be safe and make it very simple to filter out all the original files that can be deleted after all cutting is done.



There could even be a confirmation dialog asking "Are you sure you want to rename the file "filename.mpg" to filename_full.mpg"  with yes as the default so "Enter" could be user to confirm the "yes" and "Esc" to cancel.


For me this is a feature i missed straight away when using the application.
I'm sure there are other users who would use this once implemented.

It is basically a "i have what i need from this file command."

**Comment by reefhound** [comment]
maybe unfair but mifi will get blamed when user deletes their originals by mistake by clicking on a button in the app. What I do is create my clips, and for a file I want to keep entirely, I create a clip from 0 to maybe a frame before the end. Then there will be a "clip" file with timepoints for any video I want to keep. I can then delete all original files. While you may lose a frame at the end (or beginning) this might achieve what you seek in the workflow process.

**Comment by svefro** [comment]
If the file is 4gb and i do it your way i have to write 4 gigs to the disk.
That takes a while. And the feature should have a comfirmation dialog. with
default comfirm. users who click without reading diserves a file delete. :p

**Comment by mifi** [comment]
I have implemented a "move to trash" function with a confirmation. I feel that's quite safe.
Any suggestions for hotkey? Should try to use something that's common in other editors. Will release soon when i have tested

**Comment by mifi** [comment]
Now released 1.14.0 with trash button. Does this cover your use case @svefro  ?

**Comment by svefro** [comment]
Very nice @mifi 👍 a hotkey is always nice. Can the "Del" key be used?

**Comment by mifi** [comment]
Closing this in favor of #114
